WalletWasabi.Documentation/Guides/WasabiCodingGuidelines.md

```diff
@@ -0,0 +1,272 @@
+		}
+
+		public bool MyConfirmed
+		{
+			get => _myConfirmed;
+			set => this.RaiseAndSetIfChanged(ref _myConfirmed, value);
+		}
+
+		public ReactiveCommand MyCommand { get; }
+		public ReactiveCommand MySecondCommand { get; }
+
+		public MyViewModel()
+		{
+
+			this.WhenAnyValue(x => x.MyProperty)
+				.Subscribe(myProperty =>
+				{
+				});
+
+			_confirmed = Model.WhenAnyValue(x => x.Confirmed)
+				.ToProperty(this, x => x.MyConfirmed, scheduler: RxApp.MainThreadScheduler)
+				.DisposeWith(Disposables);
+
+			Model.WhenAnyValue(x => x.IsBanned, x => x.SpentAccordingToBackend)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(_ =>
+				{
+				})
+				.DisposeWith(Disposables);
+
+			Dispatcher.UIThread.PostLogException(() =>
+			{
+				//do something on the UI thread. App-crash exception handler built-in.
+			});
+
+			Observable.FromEventPattern(
+				Global.ChaumianClient,
+				nameof(Global.ChaumianClient.StateUpdated))
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(_ =>
+				{
+				}).DisposeWith(Disposables);
+
+			IObservable<bool> isCoinListItemSelected = this.WhenAnyValue(x => x.MyProperty).Select(myProperty => myProperty != null);
+
+			MyCommand = ReactiveCommand.Create(() =>
+			{
+
+			}, isCoinListItemSelected);
+
+			MyCommand.ThrownExceptions
+				.Subscribe(ex => Logging.Logger.LogWarning<MyViewModel>(ex));
+
+			MySecondCommand = ReactiveCommand.CreateFromTask(async () =>
+			{
+				await Task.Delay(100);
+			}, isCoinListItemSelected);
+
+			Observable.FromEventPattern(Global.WalletService.Coins, nameof(Global.WalletService.Coins.CollectionChanged))
+				.Merge(Observable.FromEventPattern(Global.WalletService, nameof(Global.WalletService.NewBlockProcessed)))
+				.Merge(Observable.FromEventPattern(Global.WalletService, nameof(Global.WalletService.CoinSpentOrSpenderConfirmed)))
+				.Throttle(TimeSpan.FromSeconds(5))									
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(_ => 
+				{
+
+				})
+				.DisposeWith(Disposables);
+
+
+		}
+
+		#region IDisposable Support
+
+		private volatile bool _disposedValue = false;
+
+		protected virtual void Dispose(bool disposing)
+		{
+			if (!_disposedValue)
+			{
+				if (disposing)
+				{
+					Disposables?.Dispose();
+				}
+				Disposables = null;
+
+				_disposedValue = true;
+			}
+		}
+
+		~MyViewModel()
+		{
+			Dispose(false);
+		}
+
+		public void Dispose()
+		{
+			Dispose(true);
+			GC.SuppressFinalize(this);
+		}
+
+		#endregion IDisposable Support
+	}
+```
+      
+      
+## Detecting the change of a Property
+
+You can put the following into the contructor of the ViewModel. MyProperty should use __this.RaiseAndSetIfChanged__ to notify the change. If you're a VM and you e.g. WhenAnyValue against your own property, there's [no need to clean that up because](https://reactiveui.net/docs/guidelines/framework/dispose-your-subscriptions) that is manifested as the VM having a reference to itself.
```

Comment:
MyProperty is code. Use code blocks, like this: `MyProperty`

Reply:
Same applies to this.RaiseAndSetIfChanged -> `this.RaiseAndSetIfChanged`. I guess you were looking for this, but you didn't know about it and you decided to do bold instead. Please fix it consistently everywhere.